pyproject.toml

```diff
@@ -27,25 +27,21 @@ seaborn = "^0.11.1"
 scipy = "^1.7.3"
 scikit-learn = ">=0.24,<1.1"
 japanmap = ">=0.0.21,<0.0.25"
-better-exceptions = "^0.3.2"
 requests = "^2.25.1"
 ruptures = "^1.1.1"
-tabulate = "^0.8.7"
```

Comment:
Please consider to include `tablate` as commented on #1161.

Reply:
Done.